wp-content/themes/chrisdinh-photography/resources/views/front-page.blade.php

```diff
@@ -9,6 +13,8 @@
 
   @unless ( empty($slides) && empty($sliderSettings) )
     @include('components.slider', ['data' => $slides, 'settings' => $sliderSettings])
+    @if (get_field('slider__show-camera-effect'))
```

Comment:
Use `$sliderSettings` instead

Reply:
updated in commit 1569f54c28e33513007525a18dde9d946fba34c1